src/Components/Auth/Login.tsx

```diff
@@ -95,15 +100,15 @@ export const Login = (props: { forgot?: boolean }) => {
     const valid = validateData();
     if (valid) {
       // replaces button with spinner
-      setLoading(true);
+      setLoading(queryLoading);
 
-      dispatch(postLogin(valid)).then((resp: any) => {
+      refetch().then((resp: any) => {
```

Comment:
You'll have to use the `request` method to dispatch this instead of invoking `refetch`

Reply:
Okay got it